Index.md

```diff
@@ -44,7 +44,7 @@
 | [Quick Sort](https://github.com/jainaman224/Algo_Ds_Notes/tree/master/Quick_Sort) | :heavy_multiplication_x: | [:heavy_check_mark:](https://github.com/jainaman224/Algo_Ds_Notes/blob/master/Quick_Sort/Quick_Sort.coffee) | [:heavy_check_mark:](https://github.com/jainaman224/Algo_Ds_Notes/blob/master/Quick_Sort/Quick_Sort.cpp) | :heavy_multiplication_x: | [:heavy_check_mark:](https://github.com/jainaman224/Algo_Ds_Notes/blob/master/Quick_Sort/Quick_Sort.java) | :heavy_multiplication_x: | :heavy_multiplication_x: | [:heavy_check_mark:](https://github.com/jainaman224/Algo_Ds_Notes/blob/master/Quick_Sort/Quick_Sort.py) | :heavy_multiplication_x: |
 | [Quick Sort 3-way](https://github.com/jainaman224/Algo_Ds_Notes/tree/master/Quicksort%203-way) | :heavy_multiplication_x: | :heavy_multiplication_x: | :heavy_multiplication_x: | :heavy_multiplication_x: | :heavy_multiplication_x: | :heavy_multiplication_x: | :heavy_multiplication_x: | [:heavy_check_mark:](https://github.com/jainaman224/Algo_Ds_Notes/blob/master/Quicksort%203-way/sorting.py) | :heavy_multiplication_x: |
 | [Radix Sort](https://github.com/jainaman224/Algo_Ds_Notes/tree/master/Radix_Sort) | :heavy_multiplication_x: | [:heavy_check_mark:](https://github.com/jainaman224/Algo_Ds_Notes/blob/master/Radix_Sort/Radix_Sort.coffee) | [:heavy_check_mark:](https://github.com/jainaman224/Algo_Ds_Notes/blob/master/Radix_Sort/Radix_Sort.cpp) | :heavy_multiplication_x: | [:heavy_check_mark:](https://github.com/jainaman224/Algo_Ds_Notes/blob/master/Radix_Sort/Radix_Sort.java) | :heavy_multiplication_x: | :heavy_multiplication_x: | [:heavy_check_mark:](https://github.com/jainaman224/Algo_Ds_Notes/blob/master/Radix_Sort/Radix_Sort.py) | :heavy_multiplication_x: |
-| [Reversal Linked List](https://github.com/jainaman224/Algo_Ds_Notes/tree/master/Reversal_Linked_List) | [:heavy_check_mark:](https://github.com/jainaman224/Algo_Ds_Notes/blob/master/Reversal_Linked_List/Reversal_Linked_List.c) | :heavy_multiplication_x: | :heavy_multiplication_x: | :heavy_multiplication_x: | [:heavy_check_mark:](https://github.com/jainaman224/Algo_Ds_Notes/blob/master/Reversal_Linked_List/Reversal_Linked_List.java) | :heavy_multiplication_x: | :heavy_multiplication_x: | :heavy_multiplication_x: | :heavy_multiplication_x: |
+| [Reversal Linked List](https://github.com/jainaman224/Algo_Ds_Notes/tree/master/Reversal_Linked_List) | [:heavy_check_mark:](https://github.com/jainaman224/Algo_Ds_Notes/blob/master/Reversal_Linked_List/Reversal_Linked_List.c) | :heavy_multiplication_x: | [:heavy_check_mark:](Reversal_Linked_List/Reversal_Linked_List.cpp) | :heavy_multiplication_x: | [:heavy_check_mark:](https://github.com/jainaman224/Algo_Ds_Notes/blob/master/Reversal_Linked_List/Reversal_Linked_List.java) | :heavy_multiplication_x: | :heavy_multiplication_x: | :heavy_multiplication_x: | :heavy_multiplication_x: |
```

Comment:
@mrwonko I don't know can you provide me some links and you will next time get that implemented. Please change as of now.